person_id: person
ball_id: ball
instance_groups:
  - prefix: tutorial
    order: 0
    instances:
      - start_x: 20.75
        start_y: 1.35
        start_yaw: 0
        ball_x: 38.15
        ball_y: 8.6
        max_duration: 300 #seconds
        tutorial_duration: 45 #seconds
        is_tutorial: true
      - start_x: 26.1
        start_y: 45.45
        start_yaw: -0.0624178
        ball_x: 45.7
        ball_y: 30.85
        max_duration: 300 #seconds
        tutorial_duration: 45 #seconds
        is_tutorial: true
  - prefix: pair1
    order: -1
    instances:
      # instance 1 testing directional choices
      - start_x: 14.2
        start_y: 45.4
        start_yaw: -0.0525822
        ball_x: 38.1
        ball_y: 23.8
        max_duration: 300 #seconds
        is_tutorial: false
      #instance 3 tests navigating a big room without directional arrows
      - start_x: 1.35
        start_y: 18.9
        start_yaw: 1.5708
        ball_x: 30.85
        ball_y: 35.05
        max_duration: 300 #seconds
        is_tutorial: false
      #instance 5 tests distraction in presence of a robot
      - start_x: 34
        start_y: 45.45
        start_yaw: -0.0434503
        ball_x: 40.5
        ball_y: 1.4
        max_duration: 300 #seconds
        is_tutorial: false
  - prefix: pair2
    order: -1
    instances:
      # instance 2 (paired) experiment for experiment 1 in opposite direction
      - start_x: 46.15
        start_y: 45.6
        start_yaw: 3.09617
        ball_x: 15.95
        ball_y: 23.35
        max_duration: 300 #seconds
        is_tutorial: false
      #instance 4 (paired with 3) attempts to show that directions in big room help
      - start_x: 1.35
        start_y: 18.9
        start_yaw: 1.5708
        ball_x: 30.85
        ball_y: 35.05
        max_duration: 300 #seconds
        is_tutorial: false
      #instance 6 (paired with 5) tests distraction in absense of robot
      - start_x: 34
        start_y: 45.45
        start_yaw: -0.0434503
        ball_x: 40.5
        ball_y: 1.4
        max_duration: 300 #seconds
        is_tutorial: false

